traj_gen/PolyCoeff[] poly_coeff
int8 n_seg


